server/org_config.go

```diff
@@ -0,0 +1,187 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/influxdata/chronograf"
+)
+
+type organizationConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.OrganizationConfig
+}
+
+func newOrganizationConfigResponse(config chronograf.OrganizationConfig) *organizationConfigResponse {
+	return &organizationConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config",
+		},
+		OrganizationConfig: config,
+	}
+}
+
+type logViewerConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.LogViewerConfig
+}
+
+func newLogViewerConfigResponse(config chronograf.OrganizationConfig) *logViewerConfigResponse {
+	return &logViewerConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config/logviewer",
+		},
+		LogViewerConfig: config.LogViewer,
+	}
+}
+
+// OrganizationConfig retrieves the organization-wide config settings
+func (s *Service) OrganizationConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	orgID, ok := hasOrganizationContext(ctx)
+	if !ok {
+		Error(w, http.StatusBadRequest, "Organization not found on context", s.Logger)
+		return
+	}
+
+	config, err := s.Store.OrganizationConfig(ctx).FindOrCreate(ctx, orgID)
+	if err != nil {
+		Error(w, http.StatusBadRequest, err.Error(), s.Logger)
+		return
+	}
+
+	if config == nil {
```

Comment:
Is there a reason for being super defensive like this here?

Reply:
We followed the same pattern as with the server/config.go service methods `Config`, `AuthConfig`. Should we remove it?

Reply:
I'm inclined to get rid of it.

Reply:
this is on the noop and is consistent with all other error messages. we removed the constant for it and just wrote the error message inline

Reply:
@jaredscheib not sure I follow

Reply:
whoops, i wrote that on the wrong thread somehow -- that comment was meant to address the error message on the `noop` org config store.

we'll remove this guard here and on the other two. seems like if it's nil, the client should receive that.

Reply:
If it's nil and we didn't return an error something has gone horribly horribly wrong.